docker/docker-compose.yml

```diff
@@ -22,7 +22,7 @@ services:
     extra_hosts:
       - "host.docker.internal:host-gateway"
   yorkie:
-    image: 'yorkieteam/yorkie:0.4.4'
+    image: 'yorkieteam/yorkie:latest'
```

Comment:
oh let's not use `latest` but explicitly specify the version to prevent the issues we had this week 😫

Reply:
done in [c96757f](https://github.com/yorkie-team/yorkie-android-sdk/pull/126/commits/c96757fb98e66362fd70a971f9089746311ed0e4) 🥲

Reply:
shouldn't it be `0.4.5`?

Reply:
I thought it doesn't matter since either `0.4.5` or `0.4.4` is not working well with the current SDK.
but it should be `0.4.5` to check whether the yorkie tree works, so I changed it anyway: [2b509d0](https://github.com/yorkie-team/yorkie-android-sdk/pull/126/commits/2b509d07fe8b130e8876644b867e5a709d8afa32)